### DISCUSSION TITLE
Game betting contract: rock paper scissors

### DISCUSSION THREAD

**Discussion by heytdep** [stellar/sorobanathon#61] [General]
Hi! We are excited of submitting this one: https://github.com/xycloo/soroban-rock-paper-scissors-contract.

There you'll find the code for a rock-paper-scissors betting contract, as well as a walkthrough in the README.

**Comment by achallagundla** [comment]
Ayy! We gotta play rock paper scissors Soroban Edition on Stream! Will get back to you soon once we review the submission!

**Reply by heytdep** [reply]
that's a great idea! 😂

**Comment by heytdep** [comment]
I'm also starting to write some docs to play from the CLI and not just from tests to help anyone who wants to try it out on futurenet.

**Reply by rawritude** [reply]
Would love to make a video about it or use it on a stream if I can interact on FutureNet. Thanks @heytdep !

**Reply by heytdep** [reply]
Awesome! I'm planning on making it available on futurenet by the end of tomorrow/Wednesday

**Comment by heytdep** [comment]
@achallagundla @rawritude I have just added the documentation to play with the contract on Futurenet (https://github.com/xycloo/soroban-rock-paper-scissors-contract#futurenet)

**Comment by achallagundla** [comment]
Heya @heytdep  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1200 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
